jubatus/server/framework/server_base.cpp

```diff
@@ -47,6 +47,12 @@ std::string build_local_path(
   return path.str();
 }
 
+std::string build_id(const server_argv& a) {
```

Comment:
Use this instead.
https://github.com/jubatus/jubatus/blob/feature/save_returns_id_and_path/jubatus/server/framework/server_util.cpp#L385